core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -311,6 +311,32 @@ export class RteHelperModalComponent {
     return url !== text;
   }
 
+  validateVideoStartEnd(): boolean {
+    const tmpCustomizationArgs = this.tmpCustomizationArgs as Extract<
+      CustomizationArgsNameAndValueArray[number],
+      {
+        name: 'video_id' | 'start' | 'end' | 'autoplay';
+      }
+    >[];
+    let start: number = 0;
+    let end: number = 0;
+    for (const arg of tmpCustomizationArgs) {
+      if (arg.name === 'start') {
+        if (typeof arg.value === 'number') {
+          start = arg.value;
+        }
+      } else if (arg.name === 'end') {
+        if (typeof arg.value === 'number') {
+          end = arg.value;
+        }
+      }
+    }
+    if (start === 0 && end === 0) {
+      return false;
+    }
+    return start >= end;
```

Comment:
It is a bit weird that the name of the function is called `validateVideoStartEnd` but the `true` return value means this is an invalid video

Reply:
Thanks, I agree and changed the function name to isVideoStartTimeValid that returns true if the start time < end time